Automattic/wp-calypso
# Fallback to non-adjusted time if site offset is unavailable

#### Changes proposed in this Pull Request

* Allows fallback to non-adjusted times in the Activity Log if a site offset is unavailable
  * Currently if the site doesn't respond to a settings call to pull offset the Activity Log fails to render
  * This was noticed due to a `/settings` api call failing because it isn't supported yet outside Jetpack-the-plugin but could have other situations it may occur.

#### Testing instructions



* Install only the Jetpack Backup plugin on a site without Jetpack-the-plugin
* Ensure a backup or some site activity has occurred
* Visit the Activity Log for the site on cloud.jetpack.com and compare to jetpack.cloud.localhost with this patch applied

Before:
<img width="1146" alt="Screen Shot 2021-07-28 at 12 50 31 PM" src="https://user-images.githubusercontent.com/1992658/127363747-b3f66a64-50bc-4778-845b-4575912c87e7.png">

After:
<img width="1154" alt="Screen Shot 2021-07-28 at 12 50 56 PM" src="https://user-images.githubusercontent.com/1992658/127363791-f0f4c95b-355f-432b-85fd-10eac1c8253e.png">
